src/main/java/games/strategy/engine/lobby/server/db/BadWordController.java

```diff
@@ -56,15 +56,14 @@ void removeBannedWord(final String word) {
     final String sql = "select word from bad_words";
     final Connection con = Database.getDerbyConnection();
```

Comment:
Instead of writing this to basically method in this class, just make sure to apply this changes to every Connection object you can find (usages of DbUtil#closeConnection should pretty mutch get you there)